### PR TITLE
extras: time [#21]

### PR DESCRIPTION
**Pull request by smostovoy**
#21

### CODE REVIEW

File: lib/reality/extras/time.rb [hunk 1]
```diff
@@ -0,0 +1,72 @@
+require 'timezone'
+Timezone::Configure.begin do |c|
```
Comment (smostovoy) [inline lib/reality/extras/time.rb]: not sure where to keep such things

File: lib/reality/extras/time.rb [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+        include EntityTime
+
+        def time
+          @time ||= capital.time
+        end
+      end
+
+      def self.included(_mod)
+        Reality::City.include CityTime
+        Reality::Country.include CountryTime
```
Comment (zverok) [inline lib/reality/extras/time.rb]: I'd like to have mod::City and mod::Country here.

File: spec/reality/extras/time_spec.rb [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+      VCR.use_cassette('time-Kiev'){
+        @time = Time.new(coords: [50.45, 30.52])
+      }
+    }
+
+    subject {@time}
+
+    describe '#now' do
+      it 'returns current time' do
+        expect(subject.now.to_i).to be_within(2).of(::Time.now.to_i)
```
Comment (zverok) [inline spec/reality/extras/time_spec.rb]: So, for everybody not in Europe/Kyiv timezone this test will fail?..

Comment (smostovoy) [inline spec/reality/extras/time_spec.rb]: no, timestamp is not changed

File: lib/reality/extras/time.rb (hunk 2 above)
Comment (smostovoy) [inline lib/reality/extras/time.rb]: don't understand

File: spec/reality/extras/time_spec.rb (hunk 3 above)
Comment (zverok) [inline spec/reality/extras/time_spec.rb]: Ah, sorry, disregard that, forgot `to_i` is timezone-agnostic.

File: lib/reality/extras/time.rb (hunk 2 above)
Comment (zverok) [inline lib/reality/extras/time.rb]: I meant:
* if you do `Reality.include Reality::Extras::Time` it changes `Reality::Country`;
* but if you do `SomeAnotherModule.include Reality::Extras::Time` it ...nevertheless changes `Reality::Country`! Which is, like, does entire idea of using `include` useless.

**Pull request merged**